src/app/presales/components/pre-sale-widget/use-rewards.tsx

```diff
@@ -14,13 +17,15 @@ export function usePrtRewards(address: string | undefined) {
     }
```

Comment:
Do we need a check for undefined `tokenAddress` here as well?

Reply:
@janndriessen added a guard, though I think it needs to be set for a lot of other parts to work